src/main/core/Resources/config/services/command.yml

```diff
@@ -133,3 +133,23 @@ services:
             - '@doctrine.orm.entity_manager'
         tags:
             - { name: 'console.command', command: 'claroline:rich_texts:update' }
+
+    # Messenger
+
+    Claroline\CoreBundle\Command\Messenger\EnableAsyncCommand:
+        arguments:
+            - '@Claroline\CoreBundle\Library\Configuration\PlatformConfigurationHandler'
+        tags:
+            - { name: 'console.command', command: 'claroline:messenger:enable' }
+
+    Claroline\CoreBundle\Command\Messenger\DisableAsyncCommand:
+        arguments:
+            - '@Claroline\CoreBundle\Library\Configuration\PlatformConfigurationHandler'
+        tags:
+            - { name: 'console.command', command: 'claroline:messenger:disable' }
+
+    Claroline\CoreBundle\Command\Messenger\MessengerTransportCommand:
```

Comment:
IMO, these commands are not needed (we don't write command to configure other platform options).

Reply:
Command removed